go.mod

```diff
@@ -1,13 +1,12 @@
 module github.com/container-orchestrated-devices/container-device-interface
 
-go 1.17
+go 1.19
```

Comment:
Question: How does this affect consumers?

Reply:
Well, it forces them to use the same version. And that's a good point you make. I assumed all of our major downstream projects have switched to go 1.19 by now, but now that I checked it's only kubernetes and cri-o. containerd is still at 1.18. So I guess we need to switch to that instead until containerd makes the bump.

Reply:
In the `container-toolkit` (where we also consume this) we currently use `1.17`. I don't think we're blocked on increasing this though.